python-sdk/src/astro/__init__.py

```diff
@@ -1,6 +1,6 @@
 """A decorator that allows users to run SQL queries natively in Airflow."""
 
-__version__ = "1.2.0"
+__version__ = "1.3.0dev"
```

Comment:
it should be 1.3.0 or 1.2.1?

Reply:
1.3.0 -- Main should always point to the next minor